client-linuxapp/src/main.rs

```diff
@@ -75,11 +81,15 @@ async fn get_client(rv_info: Vec<RendezvousInterpretedDirective>) -> Result<Serv
         if rv_entry.user_input {
             todo!();
         }
-        return Ok(fdo_http_wrapper::client::ServiceClient::new(
-            &urls.first().unwrap(),
-        ));
+        for url in &urls {
+            service_client_list.push(fdo_http_wrapper::client::ServiceClient::new(&url));
+        }
     }
-    bail!("No rendezvous entries found we can construct a client for");
+
+    // THIS IS JUST FOR TEST, REMOVE LATER
+    service_client_list.reverse();
```

Comment:
This is just for reviewers to see what would happen if the client encounters a non-reachable rendezvous server. Once the review is approved, I'll get rid of this.

Reply:
Could you get rid of this, since otherwise the automation will merge this in?
You could just swap around the two RV entries to demonstrate this.

Reply:
Swapping the entries doesn't work, seems to be a bug probably caused by https://github.com/fedora-iot/fido-device-onboard-rs/blob/main/owner-tool/src/main.rs#L813. In any case, just deleted it and pushed the change.